app/views/tasks/_form.html.erb

```diff
@@ -0,0 +1,19 @@
+<%= form_for @task, url: url do |f| %>
+    <span class="name_form">
+        <%= f.label :name, 'Task' %>
+        <%= f.text_field :name, required: true, autofocus: true, value: name_value %>
```

Comment:
Because you are using `form_for @task` you should be able to skip the `value: name_value` part of this (and the fields below). Rails should select the value from the `@task` provided.